src/client/src/orangehrmCoreOAuthPlugin/pages/registerOAuthClient/SaveOAuthClient.vue

```diff
@@ -102,14 +102,18 @@ import {
   required,
   shouldNotExceedCharLength,
 } from '@ohrm/core/util/validation/rules';
+import {OxdSwitchInput} from '@ohrm/oxd';
 
 const initialOAuthClient = {
-  clientId: '',
-  clientSecret: '',
+  name: '',
   redirectUri: '',
+  enabled: false,
```

Comment:
```suggestion
  enabled: true,
```
will keep enable switch on by default